src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/TestAuthController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Authorization.CommitmentPermissions.Options;
+using SFA.DAS.Authorization.EmployerUserRoles.Options;
+using SFA.DAS.Authorization.Mvc.Attributes;
+using SFA.DAS.EmployerCommitmentsV2.Web.Requests;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Controllers
+{
+    [Route("test")]
+    public class TestAuthController : Controller
+    {
+        [Route("account/{AccountHashedId}/cohort/{CohortReference}")]
+        [DasAuthorize(CommitmentOperation.AccessCohort)]
+        public IActionResult AccessTest(TestRequest request)
+        {
+            return View("Index");
+        }
+
+        [Route("account/{AccountHashedId}")]
+        [DasAuthorize(EmployerUserRole.Owner)]
+        public IActionResult RoleTest(TestRequest request)
+        {
+            return View("Index");
+        }
+
+        [Route("noaccount")]
```

Comment:
I think this action might need a `DasAuthorize` attribute?

Reply:
No it's a test to make sue DasAuthorize is not called